ideascube/static/ideascube/main.css

```diff
@@ -737,6 +737,71 @@ img.cover {
     color: #C0392B;
 }
 
+.wifi_list {
+    border: 1px solid #ddd;
+    padding: 0;
+    margin: 0;
+}
+.wifi_network {
+    text-decoration: none;
+    display: block;
+    padding: 1em;
+    border-bottom: 1px dotted #ddd;
+}
+.wifi_network:hover {
+    background-color: #f1f1f1;
+    cursor: pointer;
+}
+.wifi_network span {
+    margin: auto 1em;
+}
+.right {
+    float: right;
+}
+
+.overlay {
```

Comment:
Just to be sure, you don't mean the `wifi_*` (which I need to rename `wifi-*` :stuck_out_tongue_winking_eye:) classes, only the `right`, `overlay`, and all the popup-related stuff?